proknow-sdk-test/UploadTest/UploadsTest.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Dicom;
+﻿using FellowOakDicom;
```

Comment:
this doesn't officially support .net 8.x I don't think.  does that matter?

Reply:
I don't think it matters since it still works. I'd be slightly more concerned if FellowOakDicom wasn't only used in the test project